scripts/exglobal_diag.sh

```diff
@@ -233,8 +233,12 @@ EOFdiag
    rlist="conv_gps conv_ps conv_pw conv_q conv_sst conv_t conv_uv saphir"
    for rtype in $rlist; do
       set +e
-      ${CHGRP_CMD} *${rtype}*
-      ${STRICT_ON:-set -e}
+      for rfile in *"${rtype}"*; do 
+         if [[ -s "${rfile}" ]]; then
+            ${CHGRP_CMD} "${rfile}"
+            ${STRICT_ON:-set -e}
```

Comment:
Shouldn't this be out of the `for` loop.
If there are no files that match the `*"${rtype}"` or if the `if` condition is not satisfied, `set +e` will prevail at the end of the for loop.  `set -e` should be set there based on how it is in `develop`.

Reply:
Now that we are checking for the file's existence, I'm not sure `set +e` is needed anymore at all.

Reply:
Please confirm that my update makes the correction(s) that both of you are suggesting.